.github/workflows/publish_pypi.yml

```diff
@@ -82,42 +77,35 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.12.3
-
       - name: Clone pybind11 repo (no history)
-        run: git clone --depth 1 --branch v2.10.4 https://github.com/pybind/pybind11.git
+        run: git clone --depth 1 --branch v2.11.1 https://github.com/pybind/pybind11.git
 
-      - name: Install SUNDIALS on macOS
+      # sometimes gfortran cannot be found, so reinstall gcc just to be sure
```

Comment:
Makes sense, updating this. I am reinstalling it here because `gcc` is actually available on GitHub Actions runners by default (but sometimes with an erratic configuration of `gfortran`) and Homebrew refuses to install a package if it already exists. `openblas`, `libomp`, and `graphviz` do not come pre-installed so they should be installed rather than reinstalled